include/lib/aarch64/arch_helpers.h

```diff
@@ -273,6 +273,7 @@ DEFINE_SYSREG_RW_FUNCS(cntvoff_el2)
 
 DEFINE_SYSREG_RW_FUNCS(vpidr_el2)
 DEFINE_SYSREG_RW_FUNCS(vmpidr_el2)
+DEFINE_SYSREG_RW_FUNCS(cntp_ctl_el0)
```

Comment:
We usually prefer generic changes to happen in a separate patch to platform changes, but I'll let this one go.